tests/unittests/unittests.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
I would put this into `python/podio/test_CodeGen.py` (or something similar that get's automatically picked up by the `python -m unittest discover` we run in there). That should also automatically fix the issues with the test environment which makes some tests fail at the moment.